packages/@react-spectrum/icon/docs/custom-icons.mdx

```diff
@@ -0,0 +1,105 @@
+{/* Copyright 2020 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/icon';
+import {HeaderInfo, PropTable} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/icon/package.json';
+
+```jsx import
+import {Flex} from '@react-spectrum/layout';
+import {Icon} from '@react-spectrum/icon';
+```
+
+---
+category: Icons
+---
+
+# Custom Icons
+
+Custom icons can be used by wrapping them in an Icon component.
```

Comment:
Maybe? `Custom icons are graphical metaphors or symbols that can be used to compliment existing experiences, which can be implemented by wrapping an SVG in an Icon component.`

Reply:
That description is specific to workflow icons, so I'm not sure it would apply to all custom icons.